docs/stable/store/installation_with_rocm.md

```diff
@@ -2,18 +2,74 @@
 sidebar_position: 1
 ---
 
-# Installation with ROCm (Experimental)
-
-## Latest Tested Version
-+ v0.5.1
+# Installation with ROCm
 
 ## Tested Hardware
 + OS: Ubuntu 22.04
 + ROCm: 6.2
 + PyTorch: 2.3.0
-+ GPU: MI100s (gfx908), MI200s (gfx90a)
++ GPU: MI100s (gfx908), MI200s (gfx90a), Radeon RX 7900 XTX (gfx1100)
+
+## Option 1: Build and run with docker compose (recommended)
+
+### Step 1: Clone the ServerlessLLM Repository
+
+If you haven't already, clone the ServerlessLLM repository:
+
+```bash
+git clone https://github.com/ServerlessLLM/ServerlessLLM.git
+cd ServerlessLLM/examples/docker/
+```
+
+### Step 2:  Configuration
+
+Create a directory on your host machine where models will be stored and set the MODEL_FOLDER environment variable to point to this directory:
+
+```bash
+export MODEL_FOLDER=/path/to/your/models
+```
+
+Replace /path/to/your/models with the actual path where you want to store the models.
+
+### Step 3: Start the Services
+
+Start the ServerlessLLM services using docker compose:
+
+```bash
+docker compose -f docker-compose-amd.yml up -d --build
+```
+
+This command will start a container defined in `docker-compose-amd.yml` file. The `sllm-store-server` has already been started in the container with model storage path set to `$STORAGE_PATH`. The `$STORAGE_PATH` is mounted to the host machine's `$MODEL_FOLDER`.
+
+### Step 4: Enter the container and use ServerlessLLM Store
+
+```bash
+docker exec -it sllm_store_rocm bash
+```
+
+After entering the container, you can use our scripts to save and load models. For example, you can save and load the model in transformers library:
+
+``` bash
+python3 examples/sllm_store/save_transformers_model.py --model_name facebook/opt-1.3b --storage_path $MODEL_FOLDER
+python3 examples/sllm_store/load_transformers_model.py --model_name facebook/opt-1.3b --storage_path $MODEL_FOLDER
+```
+
+Or you can also try to use our integration with vLLM:
+:::tip
+In the `Dockerfile.rocm`, we have already built the vLLM v0.5.0.post1 from source and applied our patch to the installed vLLM library. If you face any issues, you may change the content to better build the vLLM from source.
+
+For users with gfx1100 (Radeon RDNA3) GPUs, you may need to set the environment variable `VLLM_USE_TRITON_FLASH_ATTN=0` to avoid the issue that vLLM cannot be loaded. This issue is because the flash attention support for AMD GPUs currently does not support gfx1100 GPUs. For more information, you may check this [issue](https://github.com/vllm-project/vllm/issues/4514).
+``` bash
+export VLLM_USE_TRITON_FLASH_ATTN=0
+```
+:::
+
+``` bash
+python3 examples/sllm_store/save_vllm_model.py --model_name facebook/opt-1.3b --storage_path ./models
```

Comment:
Let's use "/models" for consistency

Reply:
The command seems to hang and never returns. Here’s a screenshot for reference:

<img width="1277" alt="image" src="https://github.com/user-attachments/assets/d9700594-7280-4613-b38d-4aa44aef71ed" />

the transformers code works for me.